lib/sprockets/sass_importer.rb

```diff
@@ -1,2 +1,9 @@
-# Deprecated: Require sprockets/sass_processor instead
-require 'sprockets/sass_processor'
+
+require 'sprockets/autoload'
+
+module Sprockets
+  class SassImporter < Autoload::Sass::Importers::Filesystem
+    # NOTE: Hack to support sourcemap generation in Sass-3.3
+    def public_url(a, b);'';end
```

Comment:
It seems that sass-3.3 strictly requires the `#public_url` method and throws an exception if it is not present (https://github.com/sass/sass/blob/3.3.0/lib/sass/engine.rb#L363), while sass-3.4 doesn't check its presence (https://github.com/sass/sass/blob/3.4.0/lib/sass/engine.rb#L341).

You might remember that for 3.4 the tests where passing without this hack.